ci/package.sh

```diff
@@ -11,32 +11,42 @@ mkdir -p packages
 
 
 case "$OS" in
-    'centos:7')
+    'centos:7'|'platform:el8')
         case "$ARCH" in
             'arm32v7'|'aarch64')
-                echo 'Cross-compilation on CentOS 7 is not supported' >&2
+                echo "Cross-compilation on $OS is not supported" >&2
```

Comment:
It's not supported on EL8 ?

Reply:
I haven't done an exhaustive search, but didn't readily see packages for EL8 to set up cross-compile.  I saw this [toolchain from ARM](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-a/downloads), but am punting on deeper investigation due to personal time constraints.

Reply:
If you haven't investigated, then https://github.com/Azure/iot-identity-service/pull/273/files#diff-bff6c5a7caec352e17657fc581f0a2c86581122c44e2501d2bb8278d8c5a9c98R133 needs to be changed.

Reply:
Reverted to the prior language with an explanation of the current state of investigation for RHEL 8.

Reply:
Ok, I've reverted to the prior language with another line describing the current state of investigation with RHEL.